packages/core/examples/editableTextExample.tsx

```diff
@@ -78,15 +61,20 @@ export class EditableTextExample extends BaseExample<IEditableTextExampleState>
         return [
             [
                 <IntentSelect intent={this.state.intent} key="intent" onChange={this.handleIntentChange} />,
-                <label className={Classes.LABEL} key="maxlength">
-                    Max length
-                    <input
-                        className={Classes.INPUT}
+                <div className={Classes.FORM_GROUP} key="maxlength">
+                    <label className={Classes.LABEL} htmlFor={INPUT_ID}>Max length</label>
+                    <NumericInput
+                        id={INPUT_ID}
+                        className={Classes.FORM_CONTENT}
```

Comment:
Can you apply `pt-fill` to the numeric input to ensure it has the same width as the intent dropdown? Right now, it's deferring to the default input width, which is too wide.

![image](https://cloud.githubusercontent.com/assets/443450/25864474/67754b5c-34a4-11e7-9a09-af14da0c729a.png)